authz/audit/stdout/stdout_logger.go

```diff
@@ -0,0 +1,104 @@
+/*
+ *
+ * Copyright 2023 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package stdout contains a Stdout implementation of audit logging interface
+// The Stdoutlogger prints the audit.Event using log.go in json format
+package stdout
+
+import (
+	"encoding/json"
+	"log"
+	"time"
+
+	"google.golang.org/grpc/authz/audit"
+	"google.golang.org/grpc/grpclog"
+)
+
+var grpcLogger = grpclog.Component("authz-audit")
+
+type stdoutEvent struct {
+	FullMethodName string `json:"fullMethodName"`
+	Principal      string `json:"principal"`
+	PolicyName     string `json:"policyName"`
+	MatchedRule    string `json:"matchedRule"`
+	Authorized     bool   `json:"authorized"`
+	//Timestamp is populated using time.Now() during StdoutLogger.Log call
+	Timestamp string `json:"timestamp"`
+}
+
+// StdoutLogger is an impl of audit.Logger which
+// prints the audit.Event using log.go in json format
+type StdoutLogger struct {
+}
+
+// Log implements audit.Logger.Log
+// and prints the audit.Event using log.go in json format
+func (logger *StdoutLogger) Log(event *audit.Event) {
+	jsonBytes, err := json.Marshal(convertEvent(event))
```

Comment:
Is using `MarshalIndent` an option here for nicer json formatted output? See: https://pkg.go.dev/encoding/json#MarshalIndent

This would also make test output for easily constructable and readable.

Reply:
As we discussed, json should be printed as a one-liner by design